schema/frogbot-schema.json

```diff
@@ -64,12 +64,26 @@
       "pullRequestTitleTemplate": {
         "type": "string",
         "default": "",
+        "description": "Add a title to the pull request comments that have been generated by Frogbot pull request scanning.",
```

Comment:
Add a title to the pull request comments that have been generated by Frogbot pull request scanning.
-->
Add a title to pull request comments enerated by Frogbot.

Let's also apply this text to all workflows.